### PR TITLE
pkg/controller: fix staticcheck warning

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

fix staticcheck warning

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by hwdef** [comment]
@xichengliudui 
Thanks for your review.

File: pkg/controller/endpointslice/endpointset.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func (s endpointSet) Has(item *discovery.Endpoint) bool {
 
 // Returns an endpoint matching the hash if contained in the set.
 func (s endpointSet) Get(item *discovery.Endpoint) *discovery.Endpoint {
-	got, _ := s[hashEndpoint(item)]
+	got := s[hashEndpoint(item)]
```
Comment (mikedanese) [inline pkg/controller/endpointslice/endpointset.go]: return s[hashEndpoint(item)]

Comment (hwdef) [reply pkg/controller/endpointslice/endpointset.go]: Thanks for your suggestion.
I changed it.

**Pull request merged**